corehq/pillows/mappings/app_mapping.py

```diff
@@ -1,6 +1,6 @@
 from corehq.util.elastic import es_index
 
-APP_INDEX = es_index("hqapps_2016-01-05_1821")
+APP_INDEX = es_index("hqapps_2016-01-29_0013")
```

Comment:
exactly - otherwise the checkpoints will be out of whack and it'll have difficulty picking up new changes from the new db.  Ran into that problem with the domains migration